ansible/roles/monasca/templates/monasca-log-metrics/monasca-log-metrics.json.j2

```diff
@@ -0,0 +1,18 @@
+{
+    "command": "/usr/share/logstash/bin/logstash --log-in-json --log /var/log/kolla/monasca/monasca-log-metrics.log -f /etc/logstash/conf.d/log-metrics.conf",
+    "config_files": [
+        {
+            "source": "{{ container_config_directory }}/log-metrics.conf",
+            "dest": "/etc/logstash/conf.d/log-metrics.conf",
+            "owner": "logstash",
+            "perm": "0600"
+        }
+    ],
+    "permissions": [
+        {
+            "path": "/var/log/kolla/monasca",
+            "owner": "logstash:kolla",
```

Comment:
Is this going to fight with other containers setting the same directory to monasca ownership?

Reply:
Logs go to logstash folder now.